pandas/_libs/tslibs/np_datetime.pyi

```diff
@@ -0,0 +1 @@
+class OutOfBoundsDatetime(ValueError): ...
```

Comment:
again, only in pyx files

Reply:
This is imported in pandas/_libs/tslib.pyx and imported in the main code from there. Should probably update pandas/_libs/tslib.pyi (either import from here and allow re-export or just define there)